distribution/docs/src/main/resources/_contents/_running/command-scheduler-contents.adoc

```diff
@@ -20,7 +20,8 @@ Configure the Command Scheduler to execute a command at specific intervals.
 . Select the *${branding} Platform* Application.
 . Select *Platform Command Scheduler*.
 . Type the command or commands to be executed in the *Command* text field. Commands can be separated by a semicolon and will execute in order from left to right.
-. Type in a positive integer for the *Interval In Seconds* field.
+. Type in an interval in the *Interval* field.  This can either be a Cron expression or a positive integer (seconds) (e.x. '0 0 0 1/1 * ? *' or '12').
```

Comment:
As I recall, Quartz cron strings are a smidge different from *nix cron strings in that they include seconds as the first parameter. It might be good to call that out here.